sql/tables.sql

```diff
@@ -146,5 +146,34 @@ CREATE INDEX IF NOT EXISTS chunk_index_hypertable_id_hypertable_index_name_idx
 ON _timescaledb_catalog.chunk_index(hypertable_id, hypertable_index_name);
 SELECT pg_catalog.pg_extension_config_dump('_timescaledb_catalog.chunk_index', '');
 
+CREATE TABLE IF NOT EXISTS _timescaledb_catalog.bgw_job (
```

Comment:
I was just going to use sql to upsert a row in. This can be done in the sql scripts and the upgrade script. We should do this after everything is merged.